qemu/tests/cfg/virtio_fs_support_win_fs.cfg

```
@@ -0,0 +1,57 @@
+- virtio_fs_support_win_fs:
+    type = virtio_fs_support_win_fs
+    only Windows
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    kill_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = ${mem}M
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    io_timeout = 600
+    fs_dest = '/mnt/${fs_target}'
+    guest_username = "user00001"
+    guest_username_pwd = "U1pwd345?a"
+    add_user_cmd = "net user ${guest_username} ${guest_username_pwd} /add"
+    del_user_cmd = "net user ${guest_username} /delete"
+    fs_source_dir = virtio_fs_test/
+    driver_name = viofs
+    virtio_win_media_type = iso
+    cdroms += " virtio"
+    # install winfsp tool
+    i386, i686:
+        install_winfsp_path = 'C:\Program Files'
+    x86_64:
+        install_winfsp_path = 'C:\Program Files (x86)'
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+    tool_package_path = "%s:\7z-x64.exe"
+    i386:
+        tool_package_path = "%s:\7z-x86.exe"
+    7z_installation_path = "C:\7z"
+    install_7z_cmd = '%s /accepteula -u ${guest_username} -p ${guest_username_pwd}'
+    install_7z_cmd += ' %s /accepteula -u Administrator -p Kvm_autotest %s /S '
+    install_7z_cmd += '/D="${7z_installation_path}"'
+    uninstall_7z_cmd = "${7z_installation_path}\Uninstall.exe /S"
+    run_7z_cmd = "%s /accepteula -u ${guest_username} -p ${guest_username_pwd} "
+    run_7z_cmd += "${7z_installation_path}\7z.exe a -t7z "
+    run_7z_cmd += "${7z_installation_path}\files.7z *.txt"
```

```suggestion
    tool_installation_path = "C:\7z"
    install_tool_cmd = '%s /accepteula -u ${guest_username} -p ${guest_username_pwd}'
    install_tool_cmd += ' %s /accepteula -u Administrator -p Kvm_autotest %s /S '
    install_tool_cmd += '/D="${7z_installation_path}"'
    uninstall_tool_cmd = "${7z_installation_path}\Uninstall.exe /S"
    run_tool_cmd = "%s /accepteula -u ${guest_username} -p ${guest_username_pwd} "
    run_tool_cmd += "${7z_installation_path}\7z.exe a -t7z "
    run_tool_cmd += "${7z_installation_path}\files.7z *.txt"
```

As above, we could just update the parameters in the cfg if we want to extend it with other tools in the future.